tdvt/tdvt/tdvt_core.py

```diff
@@ -389,7 +416,7 @@ def compare_results(test_name, test_file, full_test_file, work):
     try:
         actual_xml = parse(actual_file).getroot()
         result.add_test_results(actual_xml, actual_file)
-    except ElementTree.ParseError as e:
+    except ParseError as e:
```

Comment:
This was causing an import error; at the top of the file we have `from defusedxml.ElementTree import parse, ParseError`